lms/djangoapps/teams/views.py

```diff
@@ -1130,11 +1130,17 @@ def get(self, request):
             except InvalidKeyError:
                 return Response(status=status.HTTP_404_NOT_FOUND)
 
-        if 'team_id' in request.query_params:
+        if 'team_id' in request.query_params and 'teamset_id' in request.query_params:
```

Comment:
Could you add `teamset_id` to the list of query parameters in this method's docstring?